internal/provider/resource/integration_aws_parameter_store.go

```diff
@@ -412,6 +405,26 @@ func (r *IntegrationAWSParameterStoreResource) Update(ctx context.Context, req r
 		}
 	}
 
+	updateIntegrationAuthRequest := infisical.UpdateIntegrationAuthRequest{
+		Integration:       infisical.IntegrationAuthTypeAwsSecretsManager,
+		IntegrationAuthId: plan.IntegrationAuthID.String(),
```

Comment:
I think for this one you have to use `.ValueString()` because `String()` returns a value not intended for handling. For instance in my testing, the string value returned by `String()` has "" in it:

<img width="839" alt="image" src="https://github.com/user-attachments/assets/7062fc18-f70c-44ff-974e-4dc62531d3f2">
<img width="909" alt="image" src="https://github.com/user-attachments/assets/1d19082a-18db-498e-9bca-314a66232b60">